components/card.css

```diff
@@ -88,20 +84,20 @@
   }
   
   /** Badge Card **/
-  .badges {
+  .card-badges {
     padding-top: 2rem;
     position: relative;
   }
   .btn-badge {
     position: relative;
   }
-  .badge {
+  .card-badge {
```

Comment:
`card-badges` and `card-badge` looks similar. you can give proper name to this

Reply:
sure